db.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+$db = [
+    'host' => 'mysql',
+    'user' => 'root',
+    'password' => 'secret',
+    'database' => 'yeticave'
+];
```

Comment:
Лучше использовать констаты best practice
 ```suggestion
$DB_HSOT = 'mysql';
$DB_USER = 'root';
$DB_PASSWORD = 'secret';
$DB_NAME = 'yeticave';
```